cli/src/commands/duplicate.rs

```diff
@@ -30,13 +37,36 @@ use crate::ui::Ui;
 
 /// Create a new change with the same content as an existing one
```

Comment:
nit: better to document the difference between the default and `-d`/`-A`/`-B` behavior?